runtime/src/main/java/org/corfudb/runtime/object/transactions/ConflictSetInfo.java

```diff
@@ -28,6 +29,12 @@
         return p.getSerializer().hash(o);
     }
 
+    public Optional<ICorfuSMRProxyInternal> getProxy(UUID stream) {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this use of "ICorfuSMRProxyInternal"; it is deprecated. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)